RELEASE.md

```diff
@@ -117,6 +120,29 @@ Promotion should occur in two steps:
 
 **NOTE**: The promotion of wheels to PyPI can only be done once so take caution when attempting to promote wheels to PyPI, (see https://github.com/pypa/warehouse/issues/726 for a discussion on potential draft releases within PyPI)
 
+# Doing a Patch Release
+
+A patch release is a maintenance release of PyTorch that includes fixes for regressions found in a previous minor release.
+
+## Things to consider for a patch release
```

Comment:
what does this mean?  Do you mean "Criteria for doing a Patch Release?"

Reply:
Yes, changing the header to `Patch Release Criteria`